### ISSUE TITLE
Accessibility: screen reader users can no longer send transactions

### ISSUE THREAD

**Issue by shortwavesurfer2009** [cake-tech/cake_wallet#2195]
**Describe the bug**
After the new send dialog was implemented, the action changed from a send button to a swipe to send, which does not have its own UI element for screen reader users, and does not have any kind of actions for screen reader users to interact with it. So we are unable to send transactions.

**To Reproduce**
1. Enable talkback from android system settings
2. Setup a send transaction
3. Press send button
4. Confirm box will appear
5. Flick from left to right with 1 finger

**Expected behavior**
I should be able to send a transaction. Either the swipe to send bar needs to show up as its own element that talkback can focus on and have an action for sending or it needs to be changed back to a send button instead.

**Screenshots**

https://files.catbox.moe/9scsq2.mp4

**Platform:**
 - OS: [e.g. iOS 15.1, Android 14] Android 14
 - Device: [e.g. iPhone 14, Galaxy S21] OnePlus Nord N200
 - Cake Wallet Version: [e.g. 4.12.1] monero.com v4.25.0

**Additional context**
